Erro na linha 1: Não pode invocar o resultado de uma invocação.
